var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -36,6 +36,7 @@ class Qt(Package):
     list_url = 'http://download.qt.io/archive/qt/'
     list_depth = 3
 
+    version('5.10.0', 'c5e275ab0ed7ee61d0f4b82cd471770d', url='https://download.qt.io/official_releases/qt/5.10/5.10.0/single/qt-everywhere-src-5.10.0.tar.xz')
```

Comment:
Trying to decide whether we should modify `url_for_version` to handle this, or just give up and get rid of `url_for_version` since they keep changing the URL with every other release...

Reply:
actually here only suffix changed `tar.xz`, so i can try using `url_for_version` for that.

Reply:
I looked at `url_for_version`, they really don't want to make life of packagers easy.... ;-(

Reply:
Nope, it's absurd. I think I wrote that `url_for_version` too.

Reply:
@adamjstewart so do you want me to substitute it by hardcoded `url=`?

Reply:
I can't decide if I'm biased because I wrote the `url_for_version` or not. It would certainly make the package file shorter. I think every release is actually available in the `official_releases` directory now. I think I'm okay with either decision as long as we're uniform.

Reply:
@adamjstewart i adjusted `url_for_version` for now.